toucan_connectors/facebook_ads/doc.md

```diff
@@ -0,0 +1,3 @@
+You will need to get a client_id and a client_secret, these can be found in your Facebook for Developers account under the Basic menu of your facebook application.
+
+The client_id value is under "App ID" and the client_secret value is under "App Secret".
```

Comment:
I tthink you should let the `{{redirect_uri}}` template. It's helpful for configuration

Reply:
Fixé ici : https://github.com/ToucanToco/toucan-connectors/pull/349/commits/05cf9cbd357446421582efbcb661d2f6e3e47a61